iota-client/tests/api_test.rs

```diff
@@ -6,7 +6,7 @@ use bee_signing::*;
 use bee_ternary::*;
 use iota_client::response::*;
 
-#[tokio::test]
+#[smol_potat::test]
```

Comment:
Could you simply explain why use `smol_potat` instead of `tokio`?

Reply:
It's new runtime lately which is compatible to all other runtime and framework. But instead of create a whole runtime by default, it create only an executor which definitely faster when we testing.
Btw, this is dev-dependency which users are not going to use. They can choose the one they fond of.